### ISSUE TITLE
Fatal Exception: NSInternalInconsistencyException

### ISSUE THREAD

**Issue by Ewg777** [Yummypets/YPImagePicker#35]
1. Open YPImagePicker. Library mode;
2. Switch to an another app;
3. Save some image to the library. 
4. Switch back to the app. Crash!
@s4cha 

> Invalid update: invalid number of items in section 0. The number of items contained in an existing section after the update (209) must be equal to the number of items contained in that section before the update (209), plus or minus the number of items inserted or deleted from that section (1 inserted, 0 deleted) and plus or minus the number of items moved into or out of that section (0 moved in, 0 moved out).

```
Fatal Exception: NSInternalInconsistencyException
0  CoreFoundation                 0x181343164 __exceptionPreprocess
1  libobjc.A.dylib                0x18058c528 objc_exception_throw
2  CoreFoundation                 0x181343038 +[NSException raise:format:]
3  Foundation                     0x181cdd7f4 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:]
4  UIKit                          0x18b399760 -[UICollectionView _endItemAnimationsWithInvalidationContext:tentativelyForReordering:animator:]
5  UIKit                          0x18b39d250 -[UICollectionView _endUpdatesWithInvalidationContext:tentativelyForReordering:animator:]
6  UIKit                          0x18b39d534 -[UICollectionView _performBatchUpdates:completion:invalidationContext:tentativelyForReordering:animator:]
7  UIKit                          0x18b39d390 -[UICollectionView _performBatchUpdates:completion:invalidationContext:tentativelyForReordering:]
8  UIKit                          0x18b39d314 -[UICollectionView _performBatchUpdates:completion:invalidationContext:]
9  UIKit                          0x18aab921c -[UICollectionView performBatchUpdates:completion:]
10 YPImagePicker                  0x100d044f8 (Missing)
11 YPImagePicker                  0x100d0419c (Missing)
12 YPImagePicker                  0x100cf28c4 (Missing)
13 libdispatch.dylib              0x180cc2a54 _dispatch_call_block_and_release
14 libdispatch.dylib              0x180cc2a14 _dispatch_client_callout
15 libdispatch.dylib              0x180ccf698 _dispatch_main_queue_callback_4CF$VARIANT$mp
16 CoreFoundation                 0x1812eb344 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
17 CoreFoundation                 0x1812e8f20 __CFRunLoopRun
18 CoreFoundation                 0x181208c58 CFRunLoopRunSpecific
19 GraphicsServices               0x1830b4f84 GSEventRunModal
20 UIKit                          0x18a9615c4 UIApplicationMain
```

**Comment by s4cha** [comment]
@Ewg777 nice catch, I will take a look tomorrow 👍

**Comment by s4cha** [comment]
@Ewg777 This should be fixed in 2.7.0
https://github.com/Yummypets/YPImagePicker/releases/tag/2.7.0
Thanks for taking the time to report the issue 👍
